src/main/java/it/pagopa/transactions/client/NodeForPspClient.java

```diff
@@ -26,6 +26,7 @@
 @Slf4j
 public class NodeForPspClient {
 
+    private static final String CLOSE_PAYMENT_CLIENT_ID = "ecomm";
```

Comment:
I would put this as configuration parameter valued with ecomm, what do you think?

Reply:
I believe that the parameter could be of little use considering that we will not change it so easily.
What do you think ? 
However, if we wanted we could parameterize it.

Reply:
Yes, maybe you're right, nevermind :D